content/en/Getting started/customize-report.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Customize Your Pentest Report"
+linkTitle: "Customize Your Report"
+weight: 310
+description: >
+  You may be able to customize your pentest report.
+---
+
+{{% pageinfo %}}
+This feature may be limited to subscribers with specific [PtaaS Tiers](https://cobaltio.zendesk.com/hc/en-us/articles/4408839706132-Cobalt-PtaaS-Tiers).
+{{% /pageinfo %}}
+
+You can simplify what's included in your pentest reports. This page describes available pentest
+report types, along with what you can do to leave out one or more sections from your reports.
+
+In our application, we make pentest reports available when they're ready for 
+[Remediation](../../penteststates/), or when they're [Closed](../../penteststates/). 
+To find and customize what you see in a report, take the following steps:
+
+1. Select **Pentests**, select the **State** dropdown box, and select **Remediation**.
+
+1. Now select your pentest, and select the **Report** tab. You'll see report sections,
+   along with a drop-down option that allows you to select from the [Pentest Report
+   Types](#pentest-report-types) shown in the linked table.
+
+   <!-- I think this is better without the screenshot   
+   ![Different Pentest Report Types](/gsg/PentestReportTypes.png "You can select different Pentest Report Types.") -->
+
+1. You can now customize the selected report type. Select **Customize**, and scroll to a
+   report section. 
+
+1. If you want to leave out a report section, select the eye icon next to the section title.
+   As we report all findings, we do not allow you to leave out any finding details.
+
+   ![Customize a pentest report](/gsg/CustomizePentestReport.png "Select the eye icon to leave a section out of a pentest report.")
+
+1. When you've finished customizing your report, scroll to the top of the page and select **Apply**. 
+
+1. Now you can select **Download** to download your pentest report, as a PDF file, with the
+   changes you configured.
+
+### Pentest Report Types
+
```

Comment:
A line here like "In addition to the customized report, you can also download one of these preset report formats" might clarify the above.